nav2_route/src/path_converter.cpp

```diff
@@ -30,9 +29,19 @@ void PathConverter::configure(nav2_util::LifecycleNode::SharedPtr node)
   nav2_util::declare_parameter_if_not_declared(
     node, "path_density", rclcpp::ParameterValue(0.05));
   density_ = static_cast<float>(node->get_parameter("path_density").as_double());
+  nav2_util::declare_parameter_if_not_declared(
+    node, "smoothing_radius", rclcpp::ParameterValue(1.0));
+  smoothing_radius_ = node->get_parameter("smoothing_radius").as_double();
+  nav2_util::declare_parameter_if_not_declared(
+    node, "angle_of_interpolation", rclcpp::ParameterValue(0.1));
```

Comment:
Would it be more natural for this to be a distance instead of an angle for parameterization? Or do you need an angle?